eng/pipelines/global-build.yml

```diff
@@ -0,0 +1,55 @@
+# The purpose of this pipeline is to exercise local developer workflow in the consolidated
+# runtime repo. In particular, it is supposed to run the root "build" script just like any
+# normal developer normally would and monitor regressions w.r.t. this fundamental scenario.
+
+trigger: none
+
+pr:
+  branches:
+    include:
+    - master
+    - release/*.*
+  paths:
+    include:
+    - '*'
+    - docs/manpages/*
+    exclude:
```

Comment:
What about excluding `eng\pipelines\coreclr`, `eng\pipelines\libraries`, `eng\pipelines\` and including `eng\pipelines\global-build.yml`?

Reply:
Fixed.